pkg/collector/systemperf_collector.go

```diff
@@ -1,18 +1,38 @@
 package collector
 
 import (
-	"github.com/Azure/aks-periscope/pkg/utils"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	restclient "k8s.io/client-go/rest"
+	metrics "k8s.io/metrics/pkg/client/clientset/versioned"
 )
 
 // SystemPerfCollector defines a SystemPerf Collector struct
 type SystemPerfCollector struct {
-	data map[string]string
+	kubeconfig *restclient.Config
+	data       map[string]string
+}
+
+type NodeMetrics struct {
```

Comment:
use camelCase for json properties

Reply:
umm? sure, how this is different to this: https://github.com/Azure/aks-periscope/blob/master/pkg/collector/helm_collector.go#L15 ?